### DISCUSSION TITLE
Delete temp folder before close the app

### DISCUSSION THREAD

**Discussion by voska7** [staxrip/staxrip#872] [Q&A] [answered]
is there any option to delete the temp folder before close the app if there is any temp folder available? also if abort any task delete the temp folder as well?

**Comment by jessielw** [comment, marked as answer]
Tools > Settings (or just hit F3), navigate to system on the left menu, select your delete temp files option there

**Reply by ahmed-abdelrazek** [reply]
that option isn't there any more

<img width="661" height="393" alt="image" src="https://github.com/user-attachments/assets/566ab2fb-1188-4802-9001-26057ef90c6b" />

**Reply by Dendraspis** [reply]
It's now inside the Project Options, section `Paths » Temp Files`

**Reply by ahmed-abdelrazek** [reply]
@Dendraspis thank you for the answser and the great project
